packages/playground-examples/copy/en/4-0/New JS Features/Nullish Coalescing.ts

```diff
@@ -0,0 +1,41 @@
+//// { compiler: { ts: "3.9.0" } }
+
+// # Nullish Coalescing
+//
+// This is a new operator `??` which is meant to augment
+// the common usage of `||` in the same way `===` augments `==`
+// as a more strict form of equality.
+//
+// To understand it, let's look a how || works:
+
+const response = {
+  nullValue: null,
+  headerText: "",
+  animationDuration: 0,
+  height: 400,
+  showSplashScreen: false,
+} as const;
+
+const undefinedValue = response.undefinedValue || "some other default";
+// This would be: 'some other default'
+
+const nullValue = response.nullValue || "some other default";
+
+// This works similar to most languages, however when you come from
+// other languages, then the loose equality check can be surprising:
```

Comment:
This wording is pretty vague.

1. "This works". What works? `||`?
2. "Loose equality". Probably better to say "Javascript's falsy checks", since I *think* that's what you mean here.
3. "Most languages"/"Other languages". Like Perl, Python, C, Ruby, Tcl, Bash, Groovy, various ancient Lisps?

Maybe something along the lines of "`||` is pretty good at defaulting things, but Javascript's falsy checks can surprise you for some common values"

Reply:
Somebody should write a history of truthiness in programming languages, that would be appropriately fascinating and trivial.

Good bachelor's thesis topic!

Reply:
Thanks, updated